Automattic/vip-go-mu-plugins
# Enables search index versioning self healing

## Description


## Changelog Description

### Search Corrupted Index Versioning self-healing

After reviewing logs from our corrupted index detection, we are confident in enabling the self-healing. This change enables the self-healing and shortens the interval between checks.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Break your versioning options by running - `lando wp eval "delete_option('vip_search_index_versions');"`
1. Run `wp vip-search index-versions list post` shows correct versions (with lowest version active)
1. Add version 
1. Remove option  `lando wp eval "delete_option('vip_search_index_versions');"`
1. Run `wp vip-search index-versions list post` shows correct versions (with lowest version active)


Note:
To clean cache lock in order to test:
`lando wp eval "wp_cache_set('index_versions_self_heal_lock', 0, 'vip_search');"`

To see the current option value:
`lando wp eval "var_dump(get_option( 'vip_search_index_versions'));"`
